arc/tests/e2e/database_inspection.py

```
@@ -0,0 +1,296 @@
+            ORDER BY execution_time DESC 
+            LIMIT 5
+        """))
+        
+        for i, row in enumerate(result, 1):
+            record = dict(zip(result.keys(), row))
+            # Parse JSON fields
+            if record.get('trajectory'):
+                if isinstance(record['trajectory'], str):
+                    try:
+                        record['trajectory'] = json.loads(record['trajectory'])
+                    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)